src/components/GeneratorHeader/GeneratorHeader.tsx

```diff
@@ -1,13 +1,28 @@
 import Link from 'next/link';
 import React from 'react';
+import cn from 'classnames';
+import { useRouter } from 'next/router';
 import { linkToGenerator, linkToSearch } from '../../helpers/links';
+import styles from './GeneratorHeader.module.scss';
 
 function GeneratorHeader() {
+    const router = useRouter();
+
     return (
         <header>
-            <nav>
-                <Link href={linkToGenerator()}>Generator</Link>
-                <Link href={linkToSearch()}>Search</Link>
+            <nav className={styles.root}>
+                <Link
+                    className={cn(router.asPath.includes('search') ? styles.inactiveTab : styles.activeTab, styles.tab)}
```

Comment:
Also shouldn't this be `/generator`?
```suggestion
                    className={cn(router.asPath.includes(linkToGenerator()) ? styles.inactiveTab : styles.activeTab, styles.tab)}
```